lib/src/solana_client.dart

```diff
@@ -57,84 +55,31 @@ class SolanaClient {
     return AccountInfoResponse.fromJson(data).result.value;
   }
 
-  Future<Map<String, dynamic>> _doTransfer(
-    SolanaWallet senderWallet,
-    String recipientAddress,
-    int amountInLamports, {
-    bool simulate = false,
-  }) async {
-    final Blockhash recentBlockhash = await getRecentBlockhash();
-
-    final instruction = s.Instruction(
-      programIdIndex: 2,
-      accountIndices: s.CompactArray.fromList([0, 1]),
-      data: s.CompactArray.fromList(
-        [
-          ...encodeInt(2, 32),
-          ...encodeInt(amountInLamports.toInt(), 64),
-        ],
-      ),
-    );
-
-    final message = s.Message(
-      header: s.MessageHeader(1, 0, 1),
-      accounts: s.CompactArray.fromList([
-        s.Address.from(senderWallet.address),
-        s.Address.from(recipientAddress),
-        s.Address.from(solanaSystemProgramID),
-      ]),
-      recentBlockhash: recentBlockhash.blockhash,
-      instructions: s.CompactArray.fromList([instruction]),
-    );
-    final serializedMessage = message.serialize();
-
-    final signature =
-        s.Signature.from(await senderWallet.sign(serializedMessage));
-
-    final transaction = s.Transaction(
-      signatures: s.CompactArray.fromList([signature]),
-      message: message,
-    );
-
-    return _client.request(
-      simulate ? 'simulateTransaction' : 'sendTransaction',
+  /// Send signed transaction [signedTx]
```

Comment:
Docstring should describe what function does + dot at the end, so smth like:

```suggestion
  /// Sends signed transaction [signedTx].
```